controllers/employee.js

```diff
@@ -2,6 +2,7 @@ const service = require('../services/employee')
 
 const getById = async (request, response, next) => {
     try {
+        console.log(request)
```

Comment:
remove
**BUT** set up the [Winston](https://github.com/winstonjs/winston) logger